chassis/core/src/main/java/com/griddynamics/jagger/master/MasterToJaasCoordinator.java

```diff
@@ -205,22 +209,39 @@ public StatusExchangeThread(String sessionCookie) {
         }
         
         public void setPendingRequestEntity() {
+            while (standBy && lastStatus == TestEnvironmentStatus.RUNNING && !runningStatusSent) {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(100);
+                } catch (InterruptedException e) {
+                    LOGGER.error("Error occurred while sleeping", e);
+                    standBy = false;
+                }
+            }
```

Comment:
looks dirty

we need at least detailed explanation in comments why it is done. I would recommend to look for more elegant solution